src/ape/exceptions.py

```diff
@@ -836,19 +839,27 @@ def _get_ape_traceback_from_tx(txn: FailedTxn) -> Optional["SourceTraceback"]:
 
 
 def _get_custom_python_traceback(
```

Comment:
@bitwise-constructs if you are ever curious how to hack Python's stack-frames to inject made-up frames pointing a lines in your Vyper / Solidity files, we do it here (only works on Vyper currently)